internal/database/database.go

```diff
@@ -0,0 +1,30 @@
+package database
+
+import (
+	"context"
+
+	"tracker/internal/types/user"
+)
+
+type Error string
+
+func (e Error) Error() string {
+	return string(e)
+}
+
+const (
+	ErrNotFound = Error("not found")
+)
```

Comment:
nit: keep constants at the top of the file

Reply:
On second thought, this is a different because the type it's based on is defined above.  This is fine as it is.